test/notebooks/testNotebooks/GettingStarted.ipynb

```diff
@@ -0,0 +1,110 @@
+{
+  "cells": [
```

Comment:
>"cells": [ [](start = 2, length = 10)

Nice. Can you rename this file to something so that if we see logs for this file we can quickly identify that it is our test and not the production NB.

Reply:
We wont see any logs here for running of the file. The errors/outputs will be saved to the file itself.
Running the file and parsing out outputs/checking for errors will be more elaborate, so pushed it to the next set of tests we may have to write.

Reply:
We will have to run the file, download it, parse the json and then validate results.